### ISSUE TITLE
交叉熵API错误

### ISSUE THREAD

**Issue by AndrewNoah** [MegEngine/MegEngine#120]
## 文档链接

https://megengine.org.cn/doc/basic/train_and_evaluation.html

## 问题描述

loss = F.cross_entropy_with_softmax(logits, batch_label)
实际1.0.0版本为loss = F.loss.cross_entropy(logits, batch_label)
但是文档没有更新

**Comment by ChaiMind** [comment]
收到你的反馈了，我们会尽快更新文档内容、
在文档的基础学习部分有列出 0.6 和 1.0 版本的 API 变动，按教程实现代码出现报错的时候可以进行对比。

**Comment by AndrewNoah** [comment]
👌
